extractor/intermediate.scala

```diff
@@ -91,7 +91,14 @@ case class API(
       else fixpoint(newInUse)
     }
 
-    val inUseNames = fixpoint(modelsInUse)
+    val recursivelyUsedModels = fixpoint(modelsInUse)
+
+    // check models forcibly included are not already used by the routes
+    val modelsIntersection = recursivelyUsedModels.intersect(modelsForciblyInUse)
+    if (!modelsIntersection.isEmpty)
+      throw new Exception(s"The following models are already used by the routes, no need to force inclusion: $modelsIntersection")
```

Comment:
Metarpheus doesn't seem to have a logging system (at least I would not know how to easily log and error with metarpheus) - the output is used to build the intermediate representation.
This would involve a major change.